linux/jdk/debian/build.gradle

```diff
@@ -109,7 +127,12 @@ task checkJdkDebian(type: Test) {
 	def product = getProduct()
 	def productVersion = getProductVersion()
 
-	environment "PACKAGE", "${product}-${productVersion}-jdk"
+	// If product is microsoft set it as msopenjdk
```

Comment:
I think this block is actually removing the suffix as $product would work?

Reply:
I'm not entirely sure what you mean here, but I also included these lines in this PR by mistake. They actually belong in [PR 648](https://github.com/adoptium/installer/pull/648), which is why that PR failed on first check. I will move these lines to that PR and try again.

Also, it this clears up confusion: Micrsoft's `product`="Microsoft" and `product_version`="17" | "11". Without these lines, the package would be named `microsoft-17-jdk`, but we need our package to be named `msopenjdk-17`|`msopenjdk-11`.